inc/api/namespace.php

```diff
@@ -713,6 +714,27 @@ function get_top_data( $start, $end, ?Filter $filter = null ) {
 
 		$thumbnail = $thumbnail_id ? wp_get_attachment_image_url( $thumbnail_id, get_available_thumbnail_size() ) : '';
 
+		if ( $post->post_type === 'wp_block' && Dashboard\is_block_thumbnails_allowed( $post->ID ) ) {
```

Comment:
```suggestion
		// Get block thumbnail from screen grab API.
		if ( $post->post_type === 'wp_block' && Dashboard\is_block_thumbnail_allowed( $post->ID ) ) {
```

Reply:
Should the block thumbnail allowed function check the post type is valid too? I don't think it's vital but would avoid the need for checking it here and make the function more specific. Then again, maybe we will use this API even for regular posts that don't support featured images or don't have them in future.